components/accordion/index.css

```diff
@@ -120,7 +120,7 @@ governing permissions and limitations under the License.
 	);
 
 	/* Divider */
-	--spectrum-accordion-divider-color: var(--spectrum-gray-300);
+	--spectrum-accordion-divider-color: var(--spectrum-gray-200);
```

Comment:
Won't this be a DIY token to get it surfaced to swc theme switch?

Reply:
(cc: @jnjosh) This is a good call-out, and just one example of a place where this will likely need to be addressed.